apps/bixarena/app/bixarena_app/model/error_handler.py

```diff
@@ -30,13 +30,13 @@ def get_battle_round_limit_message() -> str:
     Provide a user-facing message when the battle round cap is reached.
 
     Returns:
-        Markdown formatted message consistent with other error responses.
+        HTML formatted message for system message display.
     """
     message = (
-        "**You've reached the round limit for this battle.**\n\n"
+        "<strong>You've reached the round limit for this battle.</strong><br><br>"
         "Please wrap up this matchup or start a fresh battle!"
     )
-    return f"{message}\n\n_Round limit: {BATTLE_ROUND_LIMIT}_"
+    return f"{message}<br><br><em>Round limit: {BATTLE_ROUND_LIMIT}</em>"
```

Comment:
Make the system message more compact by removing the extra empty lines. The same comment may apply to other places.